### ISSUE TITLE
userId no necesario si hice Login 

### ISSUE THREAD

**Issue by kalarconvl** [IIC3103-2024-01/help#20]
Hola!! No entiendo si tengo que agregar el userId o no. Según la respuesta de [esta issue](https://github.com/IIC3103-2024-01/help/issues/12) es estrictamente necesario que reciba un userId, pero según [esta issue](https://github.com/IIC3103-2024-01/help/issues/6), si implementé login y mi programa almacena el userId loggeado, no es necesario.

**Comment by dedarritchon** [comment]
Hola @kalarconvl ,

Si tienes el userId en el front (como variable en memoria), puedes usarlo directamente para incluirlo en el body del request, de forma que no sea necesario escribirlo, pero "por detrás" sí se incluye en el body del request.

Epero que haya quedado más claro,

Saludos

**Comment by kalarconvl** [comment]
@dedarritchon  Y no se puede hacer el manejo del userId loggeado en el back?

**Comment by dedarritchon** [comment]
@kalarconvl ,

No, ya que el endpoint debe seguir las especificaciones del enunciado (recibir el userId como parámetro en el body del request).

Opcionalmente, podrías crear un endpoint adicional que permita crear posts de la forma que comentas, para ser usado en el front, pero el endpoint /posts que se prueba en el tester debe seguir al pie de la letra el enunciado.

Saludos

**Comment by kalarconvl** [comment]
@dedarritchon gracias por la idea!
